tests/unit/other.tcl

```diff
@@ -258,6 +258,22 @@ start_server {tags {"other"}} {
         lappend res [r get foo]
     } {12 12}
 
+    test {APPEND modifies the encoding from int to raw} {
+        r del foo
+        r set foo 1
+        r append foo 2
+
+        # append command undergoes a operation dbUnshareStringValue which
+        # converts the value to a sds object of encoding raw type.  
+        assert_equal 12 [r get foo]
+        assert_encoding "raw" foo
+
+        r set bar 12
+        assert_encoding "int" bar
+
+        assert_equal [r get foo] [r get bar]
+    } {} {needs:debug}
```

Comment:
Why is debug needed?

Reply:
Was using a debug command based validation. Not required anymore. Dropping it.